.github/actions/get-version/action.yml

```diff
@@ -12,5 +12,5 @@ runs:
     - id: get-version
       shell: sh
       run: |
-        version="$(grep version -m 1 -i package.json | sed 's/[ \",:version]//g')"
+        version="$(grep version -m 1 -i package.json | sed 's/[ \",:version]//g' | xargs)"
```

Comment:
This has been broken by #12380. Please, use this instead:

```sh
grep version -m 1 -i package.json | cut -d '"' -f4
```

Your fix looks like a patch over an outdated command.